app/Community/Components/UserProfileMeta.php

```diff
@@ -425,19 +425,27 @@ private function calculateAverageFinishedGames(array $userJoinedGamesAndAwards):
         return number_format($averageFinishedGames, 2, '.', '');
     }
 
-    private function calculateAveragePointsPerWeek(string $userMemberSince, int $points = 0): int
+    private function calculateAveragePointsPerWeek(User $user, bool $doesUserPreferHardcore, int $points = 0): int
```

Comment:
As best as I can tell, this is only called from the one place above. There's no reason to have `$points` be optional. If there is another case where points wasn't being provided, adding an extra parameter here would have broken it.

Reply:
I think I simply left points optional because it was also optional before my changes even though it was always provided then too.